acceptance-tests.md

```diff
@@ -46,18 +53,24 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
             2. Project ID / namespace as a uuid
             3. Requested quota attributes.
         6. The user will be able to authenticate using Keycloak and their institutional login.
+    * Acceptance tests:
+        1. For a given project named `myproject` for example, [Click here to check that the given project is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=project.openshift.io%7Ev1%7EProject&name=myproject).
 2. **Deactivate a project or resource allocation**
     * _As an administrator, I should be able to archive any project or resource allocation and release the resources associated with it back to the pool._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. A Coldfront admin can navigate to the project
             1. They can archive a project and expire all associated allocations by clicking ‘archive project’ by navigating to the project and clicking ‘archive project’.
             2. They can navigate to an allocation, set the status to “Denied”, and update the allocation
         2. Disabling an allocation will delete the associated OpenShift namespace, which differs from OpenStack behavior which simply disables the project.
+    * Acceptance tests:
+        1. For a given project named `myproject` for example, [Click here to check that the given project is no longer listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=project.openshift.io%7Ev1%7EProject&name=myproject).
 3. **Manage a project as a PI.**
     * _As a PI, I should be able to manage and share my project with others on the team, but no one except the PI and the administrator should be able to remove the project._
```

Comment:
Is it really true the admin can delete the project?  That is not in the use case.

Reply:
Description is correct.